project/Settings.scala

```diff
@@ -58,7 +58,8 @@ object Settings {
       "-Ywarn-unused:patvars",             // Warn if a variable bound in a pattern is unused.
       "-Ywarn-unused:privates",            // Warn if a private member is unused.
       "-Ywarn-value-discard",              // Warn when non-Unit expression results are unused.
-      "-Ybackend-parallelism", JRuntime.getRuntime.availableProcessors.toString // Run some tasks in parallel
+      "-Ybackend-parallelism", JRuntime.getRuntime.availableProcessors.toString, // Run some tasks in parallel
+      "-Yrangepos"
```

Comment:
👍 That gives nicer errors with LSP

Reply:
Yeah it's a big help.

Reply:
This settings adds like 10% overhad on compilation on 2.12.6, we may need to wait for 2.12.8. WDYT?

Reply:
Honestly I'm ok with that because it makes error reporting so much better, but we can turn it back off if it's noticeable for you.

Reply:
Mayb there is a fix by the time we can merge this PR